Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMerakiSyslog.yaml

```diff
@@ -0,0 +1,454 @@
+Parser:
+  Title: Network Session ASIM filtering parser for Cisco Meraki
+  Version: '1.2.1'
+  LastUpdated: June 5 2024
```

Comment:
"LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"